frontend/imports/ui/client/widgets/offermodal.js

```diff
@@ -7,10 +7,13 @@ import { formatError } from '/imports/utils/functions';
 
 import Tokens from '/imports/api/tokens';
 import { Offers, Status } from '/imports/api/offers';
-import convertToWei from '/imports/utils/conversion';
 
 import './offermodal.html';
 
+// TODO: DELETE THIS, TESTING PURPOSE
+window.Tokens = Tokens;
```

Comment:
I could leave it, but just was a dirty way to make it work. Probably the best, if we want to keep it public, would be exporting the variable. Let me know and I'll work on it.